src/main/java/mate/jdbc/model/Car.java

```diff
@@ -69,15 +69,12 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        String driversString = drivers == null
-                ? "null"
-                : drivers.stream().map(Driver::toString).collect(Collectors.joining("; "));
         return "Car{"
                 + "id=" + id
                 + ", manufacturer=" + manufacturer
                 + ", model='" + model + '\''
-                + ", drivers= {"
-                + driversString
+                + ", drivers={"
+                + drivers.stream().map(Driver::toString).collect(Collectors.joining("; "))
```

Comment:
live drivers only, you don't need use stream here

Reply:
ok